_studio/mfx_lib/vpp/src/mfx_vpp_hw.cpp

```diff
@@ -5104,6 +5111,73 @@ mfxU64 make_def_back_color_yuv(mfxU16 bit_depth)
         ;
 };
 
+mfxU64 GetBackgroundColor(const mfxVideoParam &videoParam)
+{
+    mfxU64 iBackgroundColor;
+    mfxU64 def_back_color = 0xffff000000000000;
```

Comment:
In which color format background color is programmed? Below you deal with different color formats, but use the same function and assign the same value. I guess background color is in some certain color format like RGB. If that's the case, then better to emphasize that in variable name like <color-format>_back_color.

Why it is 64-bit by the way?

Reply:
Address this comment, please: rename def_back_color to something which highlight output color format, for example, argb_back_color.